app/src/lib/backtranslator/structural_rules.js

```diff
@@ -367,7 +367,7 @@ function find_next_word(tokens, start_index) {
 	// Find the next word in the sentence (skip any notes or implicit markers)
 	const skip_filters = [
 		create_token_filter({ 'type': TOKEN_TYPE.NOTE }),
```

Comment:
```suggestion
		create_token_filter({ type: TOKEN_TYPE.NOTE }),
```

Reply:
I have been using single quotes for the filter/rule objects as I had json in mind. I'd like to keep it consistent. I'm open to discussing and rethinking that for a later PR though.